### PR TITLE
NextJSRowndProvider

### PR DESCRIPTION
**Pull request by davon-davis**
This PR incorporates `NextJSRowndProvider` as a submodule for the `@rownd/react` sdk

### CODE REVIEW

**Comment by mfmurray** [comment]
You could increase the `size-limit` in the package.json maybe to pass the check

**Comment by davon-davis** [comment]
@mfmurray I increased it to 50KB but the check still failed. Am I changing it in the right place?
"size-limit": [
    {
      "path": "dist/index.js",
      "limit": "50 KB"
    }
  ],

**Comment by mfmurray** [comment]
> @mfmurray I increased it to 50KB but the check still failed. Am I changing it in the right place? "size-limit": [ { "path": "dist/index.js", "limit": "50 KB" } ],

True. It only looks like the size is 2.73 kB. Have you tried running `npm run size` locally? That might help

File: package.json [hunk 1]
```diff
@@ -88,5 +98,9 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/rownd/react.git"
+  },
+  "dependencies": {
+    "next": "^14.2.13",
+    "rollup-preserve-directives": "^1.1.1"
```
[the remark this answers is not in the document]
Comment (davon-davis) [reply package.json]: Yes, I successfully moved it in devDependencies, and everything builds properly still.

**Review by mfmurray — APPROVED** [review]

**Pull request merged**